src/apps/api/views/competitions.py

```diff
@@ -164,3 +193,30 @@ class CompetitionCreationTaskStatusViewSet(RetrieveModelMixin, GenericViewSet):
     queryset = CompetitionCreationTaskStatus.objects.all()
     serializer_class = CompetitionCreationTaskStatusSerializer
     lookup_field = 'dataset__key'
+
+
+class CompetitionParticipantViewSet(ModelViewSet):
+    queryset = CompetitionParticipant.objects.all()
+    serializer_class = CompetitionParticipantSerializer
+    filter_backends = (DjangoFilterBackend, SearchFilter)
+    filter_fields = ('user__username', 'user__email', 'status', 'competition')
+    search_fields = ('user__username', 'user__email',)
+
+    def get_queryset(self):
+        qs = super().get_queryset()
```

Comment:
permissions checks?

Reply:
Not really sure what kind of permissions checks to throw in here?

Reply:
Well right now anyone could access participant models for `GET`, `PUT`, `DELETE`

You could make queryset filter only participants of competitions they are owner/collaborator on? Not really a "permissions check" as it is a filter on what participants you have access to editing/viewing